tensorflow_binding/setup.py

```diff
@@ -0,0 +1,113 @@
+"""setup.py script for warp-ctc TensorFlow wrapper"""
+
+from __future__ import print_function
+
+import os
+import re
+import setuptools
+import sys
+import unittest
+from setuptools.command.build_ext import build_ext as orig_build_ext
+
+# We need to import tensorflow to find where its include directory is.
+try:
+    import tensorflow as tf
+except ImportError:
+    raise RuntimeError("Tensorflow must be installed to build the tensorflow wrapper.")
+
+if "CUDA_HOME" not in os.environ:
```

Comment:
Many users might have CPU-only boxes. Might be worth considering whether this should work without a GPU.